qemu/tests/block_resize.py

```diff
@@ -6,6 +6,165 @@
 from virttest import utils_test
 from virttest import storage
 from virttest import data_dir
+from virttest.utils_windows import drive
+
+
+def get_block_size(test, session, block_cmd, block_pattern):
+    """
+    Get the disk size inside guest.
+
+    param block_cmd: run block_cmd command to get detail disk size info.
+    param block_pattern: use this pattern to filter the needed.
+
+    param return: return current block size.
+    """
+    output = session.cmd_output(block_cmd)
+    block_size = re.findall(block_pattern, output)
+    if block_size:
+        if not re.search("[a-zA-Z]", block_size[0]):
+            return int(block_size[0])
+        else:
+            return float(utils_misc.normalize_data_size(block_size[0],
+                                                        order_magnitude="B"))
+    else:
+        test.error("Can not find the block size for the device."
+                   " The output of command is: %s" % output)
+
+
+def compare_block_size(test, session, params, expected_size, current_size):
+    """
+    Compare the current block size with the expected size.
+
+    param expected_size: the expected size after block resize.
+    param current_size: the size from guest after block resize.
+    """
+    accept_ratio = float(params.get("accept_ratio", 0))
+    if (current_size <= expected_size and
+            current_size >= expected_size * (1 - accept_ratio)):
+        logging.info("Block Resizing Finished !!! \n"
+                     "Current size %s is same as the expected %s",
+                     current_size, expected_size)
+        return True
+    else:
+        test.fail("Block size get from guest is not same as expected.\n"
+                  "Reported: %s\nExpect: %s\n" % (current_size, expected_size))
+    return
```

Comment:
No need returen here, it wil not be executed.

Reply:
thanks, will update it